tests/ui/const-generics/unsized_const_params/symbol_mangling_v0_str.rs

```diff
@@ -0,0 +1,23 @@
+//@ check-pass
```

Comment:
This needs the `-Csymbol-mangling-version=v0` set to reproduce right?
```suggestion
//@ check-pass
//@ compiler-flags: -Csymbol-mangling-version=v0
```